### ISSUE TITLE
onChange should be hinted with ChangeEvent instead of FormEvent

### ISSUE THREAD

**Issue by crtl** [rpearce/react-expanding-textarea#61]
Currently onChange handler is wrongly typed with `FormEvent<HtmlTextareaElement>` instead of `ChangeEvent<HtmlTextaraeElement>` in [TextareaProps](https://github.com/rpearce/react-expanding-textarea/blob/main/source/index.tsx#L64)

`FormEvent.target` does not have `value` property and will cause error in strict environments.

```
function onChange(e: FormEvent<HtmlTextareaElement>) {
  setContent(e.target.value); //TS2339: Property 'value' does not exist on type 'EventTarget'.
}

```

Workaround is to explicitly convert type:
```
setContent((e.target as HTMLTextAreaElement).value);
// or
setContent(e.currentTarget.value);
```

**Comment by rpearce** [comment]
@ctrl thanks for reporting this. Let me see about rectifying it

**Comment by rpearce** [comment]
@ctrl How does this look? Is there anything else with this that needs doing? https://github.com/rpearce/react-expanding-textarea/pull/63

**Comment by crtl** [comment]
@rpearce Looks fine for me but honestly I have to ask how should I verify it?

Youve changed the type to ChangeEvent so it looks fine for me but I cannot use the branch in my CRA project because of loader issues which are ofc. unrelated to this issue.
Ive just used `yarn add https://github.com/rpearce/react-expanding-textarea.git#fix/change-event` to add it to my project but running start will cause:

```
./node_modules/react-expanding-textarea/source/index.tsx 13:30
Module parse failed: Unexpected token (13:30)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| import withForwardedRef from 'react-with-forwarded-ref';
| 
> export const getHeight = (rows: number, el: HTMLTextAreaElement): number => {
|   const {
|     borderBottomWidth,
```

**Comment by rpearce** [comment]
That's fair. Well, I think we'll be okay, for these reasons:

* the linters are happy
* the tests still pass
* I've seen other guides using `ChangeEvent` for these inputs / textareas, so it looks like I was just doing it wrong

Conversely, I could do a pre-bugfix-release, and you could test that, but I'm pretty confident this works.

If you don't care either way, I'll see about getting a patch out during my lunch break.

**Comment by rpearce** [comment]
This has been published under `v2.3.2`: https://www.npmjs.com/package/react-expanding-textarea/v/2.3.2
